### PR TITLE
[Core] Using struct instead of bool to easier work on FindNodalHProcess

### PR DESCRIPTION
**Pull request by loumalouomega**
Related with concern in #2943

### CODE REVIEW

File: kratos/processes/find_nodal_h_process.h [hunk 1]
```diff
@@ -46,6 +46,16 @@ namespace Kratos
...
+/**
+ * @brief This struct is used in order to identify when using the hitorical and non historical variables
+ */
+struct FindNodalH
+{
+    // Defining clearer options
+    constexpr static bool SaveAsHistoricalVariable = true;
+    constexpr static bool SaveAsNonHistoricalVariable = false;
+};
+
```
Comment (frastellini) [inline kratos/processes/find_nodal_h_process.h:58]: I think that @pooyan-dadvand suggested the direct use of 2 constexpr, not inside a struct.
Do you agree?

Comment (loumalouomega) [reply kratos/processes/find_nodal_h_process.h:58]: I tried, and does not work

Comment (frastellini) [reply kratos/processes/find_nodal_h_process.h:58]: ok then.

**Review by frastellini — APPROVED** [review]
Since the struct cannot be avoided, it is ok for me.

File: kratos/processes/find_nodal_h_process.h (hunk 1 above)
Comment (pooyan-dadvand) [reply kratos/processes/find_nodal_h_process.h:58]: Exactly, I suggest two constexpr in the same class. Why doesn't work?

Comment (loumalouomega) [reply kratos/processes/find_nodal_h_process.h:58]: Compilation error. I don't remember right now

Comment (pooyan-dadvand) [reply kratos/processes/find_nodal_h_process.h:58]: So at least name the struct as `FindNodalHSettings` so one can understand the use.

Comment (loumalouomega) [reply kratos/processes/find_nodal_h_process.h:58]: Solved in #3331

**Pull request merged**